doc/flatten.md

```diff
@@ -34,5 +34,11 @@ cli:
        cli-flatten-payload-max-poly-as-resource-prop-count: 8
        # max properties allowed from flatten of sub-class as param
        cli-flatten-payload-max-poly-as-param-prop-count: 8
+       # if you want to keep the flattened models even if they are not used
```

Comment:
>if you want to keep the flattened models even if they are not used [](start = 9, length = 66)

what's this for?

Reply:
It is a option in modelerfour. https://github.com/Azure/autorest.modelerfour/blob/master/modelerfour/readme.md#modelfour-options
For schema flatten, if the new property is not used by other schema, it will be removed.

Reply:
not quite understand, let's talk about this offline.

---
In reply to: [448987062](https://github.com/Azure/autorest.clicommon/pull/79#discussion_r448987062) [](ancestors = 448987062)